tests/ansible_collections/roles/oracle-linux-specific/tasks/stop_firewalld_workaround.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Check that the firewalld package is installed
+  shell: 'rpm -q firewalld'
+  ignore_errors: True
+  register: rpm_out
+
+- name: Stop the firewalld process
```

Comment:
@kokesak, @danmyway, it'd be better to do in the workaround what we tell the customers to do - set `CleanupModulesOnExit=no`.

Reply:
You are right, I updated the tests and the workaround so it does what we instruct customers.